.travis.yml

```diff
@@ -0,0 +1,93 @@
+language: php
+
+sudo: false
+
+env:
+  global:
+    - WP_PHPUNIT=phpunit
+
```

Comment:
Maybe it'd be good to add 
```
before_install:
  # turn off XDebug
  - phpenv config-rm xdebug.ini || return
```
to avoid [this message](https://travis-ci.org/mautic/mautic-wordpress/jobs/232055394#L173):
```
You are running composer with xdebug enabled. This has a major impact on runtime performance. See https://getcomposer.org/xdebug
```